plugins-mu/hm-stack/class-endpoint-controller.php

```diff
@@ -0,0 +1,238 @@
+<?php
+/**
+ * Build endpoint data for HM Stack data on this site.
+ *
+ * @package HMStackIntegration
+ */
+
+namespace HM_Stack;
+
+use WP_REST_Response;
+use WP_Error;
+use WP_Http;
+
+/**
+ * Class Endpoint_Controller
+ */
+class Endpoint_Controller extends \WP_REST_Controller {
+	/**
+	 * Register HM_Stack data routes for the WP REST API.
+	 *
+	 * @todo:: Add schemas to each endpoint.
+	 */
+	public function register_routes() {
+		$namespace = 'hm-stack/v1';
+
+		// Fetch all activity/alerts against this site.
+		register_rest_route( $namespace, 'activity', [
+			'methods'  => \WP_REST_Server::READABLE,
```

Comment:
Why are you `use`ing some classes but not others? I typically just `use` everything I need.

Reply:
Hadn't gotten to removing that yet. It should be `use`d for sure, was just working this out initially.